_posts/2022-11-10-release-1.15.3.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: post
+title:  "Apache Flink 1.15.3 Release Announcement"
+date: 2022-11-10T22:00:00.000Z
+categories: news
+authors:
+- danny:
```

Comment:
This looks incorrect?

Reply:
Yes, indeed, I am gonna fix it. Good catch.